crates/phactory/src/lib.rs

```diff
@@ -235,6 +235,9 @@ pub struct Phactory<Platform> {
 
     #[serde(default)]
     netconfig: Option<NetworkConfig>,
+
+    #[serde(skip)]
+    can_load_chain_state: bool,
```

Comment:
Will it be set false once the chain state is loaded?

Reply:
Yes, it will. and will be set to false after a `sync_header` has been invoked.